### PR TITLE
Some new cards

### PR DESCRIPTION
**Pull request by queueseven**

### CODE REVIEW

File: src/clj/game/cards-assets.clj [hunk 1]
```diff
@@ -365,6 +365,25 @@
...
+   (let [trash-ability (fn [type] {:req (req (seq (filter #(is-type? % type) (:hand runner))))
+                                   :prompt (str "Choose a " type " to trash")
+                                   :choices (req (filter #(is-type? % type) (:hand runner)))
+                                   :effect (effect (trash target))
+                                   :msg (msg " trash " (:title target) " from the Runner's Grip")})
+         choose-ability {:label "Trash 1 card in the runner's Grip of a named type"
+                         :once :per-turn
+                         :req (req (seq (:hand runner)))
+                         :prompt "Choose a card type"
+                         :choices ["Event" "Hardware" "Program" "Ressource"]
```
Comment (nealterrell) [inline src/clj/game/cards-assets.clj]: Typo, "Ressource"

Comment (queueseven) [inline src/clj/game/cards-assets.clj]: Argh, I mix it up every time....

File: src/clj/game/cards-operations.clj [hunk 2]
```diff
@@ -212,6 +212,15 @@
               " [Credits]")
     :effect (effect (gain :credit (count (filter #(not (empty? %)) (map #(:content (second %)) (get-remotes @state))))))}
 
+   "Election Day"
+   {:req (req (->> (get-in @state [:corp :hand])
+                   (filter #(not (= (:cid %) (:cid card))))
+                   (count)
+                   (pos?)))
+    :msg (msg "to trash all cards in HQ and draw 5 cards")
```
Comment (JoelCFC25) [inline src/clj/game/cards-operations.clj]: This `:msg` will yield:  "(Corp name) uses Election Day **to to** trash all cards in HQ and draw 5 cards"

Comment (queueseven) [inline src/clj/game/cards-operations.clj]: Contratulations Joel. You passed the test.
.
.
.
.
Fixed.

**Comment by mtgred** [comment]
👍

**Pull request merged**